core/tests/webdriverio_desktop/topicAndStoryEditor.js

```diff
@@ -382,3 +385,119 @@ describe('Chapter editor functionality', function() {
     }
   });
 });
+
+describe('Serial Chapter Launch functionality', function() {
+  var topicsAndSkillsDashboardPage = null;
+  var topicEditorPage = null;
+  var storyEditorPage = null;
+  var releaseCoordinatorPage = null;
+  var adminPage = null;
+  var storyName = 'Story 0';
+  var explorationEditorPage = null;
+  var dummyExplorationIds = [];
+  var dummyExplorationInfo = [
+    'Dummy exploration', 'Algorithms', 'Learn more about oppia', 'English'];
+  var topicName = 'Topic 0';
+  var topicUrlFragment = 'topic-zero';
+  var userEmail = 'creator@serialChapterTest.com';
+
+  var createDummyExplorations = async function(numExplorations) {
+    var ids = [];
+    for (var i = 0; i < numExplorations; i++) {
+      var info = dummyExplorationInfo.slice();
+      info[0] += i.toString();
+      if (i === 0) {
+        info.push(true);
+        await workflow.createAndPublishExploration.apply(workflow, info);
+      } else {
+        info.push(false);
+        await workflow.createAndPublishExploration.apply(workflow, info);
+      }
+      var url = await browser.getUrl();
+      var id = url.split('/')[4].replace('#', '');
+      ids.push(id);
+    }
+    return ids;
+  };
+
+  beforeAll(async function() {
+    topicsAndSkillsDashboardPage =
+      new TopicsAndSkillsDashboardPage.TopicsAndSkillsDashboardPage();
+    topicEditorPage = new TopicEditorPage.TopicEditorPage();
+    storyEditorPage = new StoryEditorPage.StoryEditorPage();
+    skillEditorPage = new SkillEditorPage.SkillEditorPage();
+    explorationEditorPage = new ExplorationEditorPage.ExplorationEditorPage();
+    explorationPlayerPage = new ExplorationPlayerPage.ExplorationPlayerPage();
+    releaseCoordinatorPage = (
+      new ReleaseCoordinatorPage.ReleaseCoordinatorPage());
+    adminPage = new AdminPage.AdminPage();
+    explorationEditorMainTab = explorationEditorPage.getMainTab();
+    await users.createAndLoginCurriculumAdminUser(
+      userEmail, 'creatorSerialChapterTest');
+
+    await adminPage.get();
+    await adminPage.addRole('creatorSerialChapterTest', 'release coordinator');
+    await releaseCoordinatorPage.getFeaturesTab();
+    var curriculumAdminSerialChapterFlag = (
+      await releaseCoordinatorPage.
+        getSerialChapterCurriculumAdminFeatureElement());
+    await releaseCoordinatorPage.enableFeature(
+      curriculumAdminSerialChapterFlag);
+
+    dummyExplorationIds = await createDummyExplorations(2);
+    var handle = await browser.getWindowHandle();
+    await topicsAndSkillsDashboardPage.get();
+    await topicsAndSkillsDashboardPage.createTopic(
+      topicName, topicUrlFragment, 'Description', false);
+    await topicEditorPage.createStory(
+      storyName, 'topicandstoryeditortwo', 'Story description',
+      Constants.TEST_SVG_PATH);
+    await general.closeCurrentTabAndSwitchTo(handle);
+    await users.logout();
+  });
+
+  it('should change the status and publish chapters serially.',
```

Comment:
Do we use `.` at teh end , please check in other place, if not, let's remove it! :)

Reply:
Done